### PR TITLE
[Add - Words] companies.txt and fullstack.txt

### PR DESCRIPTION
**Pull request by rbalet**
Add words in fullstack + update version.

For companies.txt, 
-  add: Gitlab, verdaccio
-  remove duplicate: Linkedin
-  reorder file

Also regenerate .gz files

### CODE REVIEW

File: packages/companies/companies.txt [hunk 1]
```diff
@@ -368,6 +368,7 @@ Genworth Financial, Inc.
 George Weston
 GeoTrust Global
 Gilead Sciences, Inc.
+Gitlab
```
Comment (d2s) [inline packages/companies/companies.txt]: Misspelling, the official version of the name is: `GitLab`
(with big `L` character).

File: packages/companies/companies.txt [hunk 2]
```diff
@@ -517,8 +518,8 @@ Liberty Interactive Corporation
 Liberty Mutual Holding Company Inc.
 Liberty Mutual Insurance Group
 Lincoln National Corporation
-linkedin
 LinkedIn
+linkedin
```
Comment (d2s) [inline packages/companies/companies.txt]: Duplicate item.

File: packages/fullstack/fullstack.txt [hunk 3]
```diff
@@ -3,79 +3,118 @@ autofill
...
 autoprefixer
 baseref
+behaviour
+charlist
+checkboxes
 classlist
 codelyzer
 collapsable
 compat
-cond
```
Comment (d2s) [inline packages/fullstack/fullstack.txt]: Is there reason why `cond` was removed?
Was it a misspelling of `conda` or is there another reason?

Comment (rbalet) [reply packages/fullstack/fullstack.txt]: Not at all, I'll fix that. Thx for the feedback

**Comment by Jason3S** [comment]
Thank you.

**Pull request merged**